lib/react_client.dart

```diff
@@ -1020,6 +1157,25 @@ dynamic _findDomNode(component) {
   return ReactDom.findDOMNode(component is Component ? component.jsThis : component);
 }
 
+class ReactDartContext {
+  ReactDartContext(this.Provider, this.Consumer, this._jsThis);
+  final ReactContext _jsThis;
+  final ReactJsContextComponentFactoryProxy Provider;
+  final ReactJsContextComponentFactoryProxy Consumer;
+  ReactContext get jsThis => _jsThis;
+}
+
+ReactDartContext createContext([
+  dynamic defaultValue,
+  int Function(dynamic currentValue, dynamic nextValue) calculateChangedBits,
+]) {
+  var JSContext = React.createContext(defaultValue, allowInterop((currentValue, nextValue) {
```

Comment:
I think `defaultValue` needs to be wrapped in `jsifyContext`, right? Can we add a test for that?